src/utils/theme.tsx

```diff
@@ -97,6 +97,33 @@ const themeOptions: ThemeOptions = {
       textTransform: 'none',
     },
   },
+  shadows: [
+    'none',
+    '0px 0px 0px 0px rgba(0,0,0,0.2)',
+    '0 6px 16px 0 rgba(0, 0, 0, 0.1)',
+    '0px 10px 40px -10px rgba(0,0,0,0.2)',
+    '0 8px 20px 0 rgba(68, 86, 108, 0.1)',
+    '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
+    '0 2px 4px 0 rgba(47, 84, 235, 0.15), 0 8px 16px 0 rgba(47, 84, 235, 0.15)',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
+    'none',
```

Comment:
You can probably change it to 1 line and leave a comment that material UI requires exactly 25 box shadows
```javascript
...[...Array(18)].map((_) => `none`)
```

Reply:
for some reason typescript doesn't like that approach with spread operator :thinking:

Reply:
I think that in this case, you can tell typescript explicitly that it will be of type Shadows like so:
```typescript
import { Shadows } from '@material-ui/core/styles/shadows';
...
shadows: [
    'none',
    '0px 0px 0px 0px rgba(0,0,0,0.2)',
    '0 6px 16px 0 rgba(0, 0, 0, 0.1)',
    '0px 10px 40px -10px rgba(0,0,0,0.2)',
    '0 8px 20px 0 rgba(68, 86, 108, 0.1)',
    '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
    '0 2px 4px 0 rgba(47, 84, 235, 0.15), 0 8px 16px 0 rgba(47, 84, 235, 0.15)',
    ...[...Array(18)].map((_) => `none`),
  ] as Shadows,
...
```